benchmarks/bench.sh

```diff
@@ -455,13 +464,27 @@ run_clickbench_extended() {
     $CARGO_COMMAND --bin dfbench -- clickbench  --iterations 5 --path "${DATA_DIR}/hits.parquet" --queries-path "${SCRIPT_DIR}/queries/clickbench/extended.sql" -o "${RESULTS_FILE}"
 }
 
+# Add cleanup function at the start of script
+cleanup_download() {
+    local file_to_clean="$1"
+    echo -e "\nCleaning up downloaded files..."
+    rm -f "${file_to_clean}"
+    exit 1
+}
+
+# Add trap before download starts
+trap cleanup_download INT TERM
+
 # Downloads the csv.gz files IMDB datasets from Peter Boncz's homepage(one of the JOB paper authors)
 # https://event.cwi.nl/da/job/imdb.tgz
 data_imdb() {
     local imdb_dir="${DATA_DIR}/imdb"
     local imdb_temp_gz="${imdb_dir}/imdb.tgz"
     local imdb_url="https://event.cwi.nl/da/job/imdb.tgz"
 
+    # Set trap with parameter
```

Comment:
Perhaps we can add a check for file size of the imdb.tgz file rather than just checking for its existence

```
        if [ ! -f "${imdb_dir}/imdb.tgz" ]; then
```